webapp/src/Utils/Utils.php

```diff
@@ -506,6 +510,8 @@ public static function cutString(string $str, int $size): string
 
     /**
      * Compute the LCS diff of two lines.
+     *
+     * @return array<int, bool|string>
```

Comment:
```suggestion
     * @return array{bool, string}
```

See https://phpstan.org/writing-php-code/phpdoc-types#array-shapes